pkg/scheduler/framework/plugins/defaultassigner/static.go

```diff
@@ -49,7 +49,7 @@ func (pl *StaticAssigner) Name() string {
 func (pl *StaticAssigner) Assign(ctx context.Context, sub *appsapi.Subscription, finv *appsapi.FeedInventory, availableReplicas framework.TargetClusters) (framework.TargetClusters, *framework.Status) {
 	klog.V(3).InfoS("Attempting to assign replicas to clusters",
 		"subscription", klog.KObj(sub), "clusters", availableReplicas.BindingClusters)
-	if sub.Spec.DividingScheduling == nil || sub.Spec.DividingScheduling.Type != appsapi.StaticReplicaDividingType {
+	if sub.Spec.DividingScheduling == nil || (sub.Spec.DividingScheduling.Type != appsapi.StaticReplicaDividingType && sub.Spec.DividingScheduling.Type != appsapi.DynamicReplicaDividingType) {
```

Comment:
This file is only for static scheduling. Please do not append dynamic scheduling here.